package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "Symphony",
-  "productName": "Symphony",
+  "productName": "Symphony - ODP",
   "version": "1.0.1",
-  "description": "Symphony desktop app",
+  "description": "Symphony desktop app (Foundation ODP)",
```

Comment:
Of course it would be even better if there was some way for SymphonyElectron to autodetect (or, less ideally, let the user choose) the pod they're connecting to.